configuration/configure-spdockit.md

```diff
@@ -20,6 +20,17 @@ This article explains how to configure the SPDocKit database and service before
     * Automatic Subscriptions and Alerts
     * Index Reorganization
     * Automatic Rules and Queries
-4. On the __Service Configuration__ wizard page click __Next__ to complete the Configuration Wizard and apply the changes.
 
```

Comment:
we need to mention local system somewhere. and when it will be available.

Reply:
we did.  spo-snapshots.md. I could however c/p here somewhere